setup.py

```diff
@@ -114,6 +104,11 @@ def readfile(fname):
     import wheelhouse_uploader.cmd
     cmdclass.update(vars(wheelhouse_uploader.cmd))
 
+python_2_6_backports = ''
```

Comment:
We install argparse only if Python version is 2.6. By default we don't need to install any backports, hence an empty string.